libwallet/src/slate_versions/mod.rs

```diff
@@ -80,6 +87,269 @@ impl VersionedSlate {
 			}
 		}
 	}
+
```

Comment:
The implementation here is different from the rest of the serialization that happens in grin. For serializing binary, a struct usually implements the 'Readable/Writeable' traits from grin core and implements methods there.

Reply:
ok I will change it to a trait